icons/fastapi.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>FastAPI</title><path d="M12 0C5.375 0 0 5.375 0 12c0 6.627 5.375 12 12 12 6.626 0 12-5.373 12-12 0-6.625-5.373-12-12-12zm-.624 21.62v-7.528H7.19L13.203 2.38v7.528h4.029L11.376 21.62z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>FastAPI</title><path d="M12.004.03A12.004 11.966 0 0 0 0 11.996 12.004 11.966 0 0 0 11.996 23.97 12.004 11.966 0 0 0 24 12 12.004 11.966 0 0 0 12.008.03m-.831 5.42h7.555l-7.589 5.329h5.185L5.28 18.546l2.4-5.329 1.096-2.43Z"/></svg>
```

Comment:
I'm seeing some differences in the inner shape (yours in black). Ex.
![image](https://github.com/simple-icons/simple-icons/assets/12817388/1d550c14-b1e2-42c0-9d88-7f6f58e496a8)

You could probably be able to remove safely 2 nodes in the straight line:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/c4064e2f-c211-4b31-bdd7-7eebeb9da0ac)

Reply:
How does it look now? I've overlayed it as you did and I'm no longer seeing any issues.

Reply:
![Screenshot 2024-03-18 at 11 55 24 (Inkscape)](https://github.com/simple-icons/simple-icons/assets/47499684/3cae3967-2841-4670-86a6-b3157ba1ad99)

Hmm actually I'm seeing one little spot there. I'll try to manually adjust now...

Reply:
That should do it!